test/db/archos/linux-x64/dbg_dt

```diff
@@ -0,0 +1,62 @@
+NAME=List all traces (dtl)
+FILE=bins/elf/analysis/calls_x64
+ARGS=
+CMDS=<<EOF
+dt++ 0 1 2
+dtl
+dtlq
+EOF
+EXPECT=<<EOF
+0x00000000 size=1 count=1 times=1 tag=0
+0x00000001 size=1 count=2 times=1 tag=0
+0x00000002 size=1 count=3 times=1 tag=0
```

Comment:
why is count 2 and 3?

Reply:
https://github.com/rizinorg/rizin/blob/f9e01ba07d768e5026d428046e784790ad962e7c/librz/debug/trace.c#L292

It looks like this `count` is more like an index

Reply:
:/ i guess there's something unclear in these traces.

Reply:
I think @yossizap might know better about debug tracing.

Reply:
>It looks like this count is more like an index

It's only used for debug print to show the number of instructions recorded. addr and tag are used as the index.